test/controllers/update/update.manage.trusts.tell.us.about.the.legal.entity.controller.spec.ts

```diff
@@ -142,7 +142,6 @@ mockGetApplicationData.mockReturnValue({
           reviewed_former_bos: false,
           reviewed_individuals: false,
           reviewed_legal_entities: false,
-          reviewed_trust_details: false,
```

Comment:
this was just a duplicate field that I noticed in the test object (if you expand the code window you can see it was in there twice)